plugins/logs/plugin.go

```diff
@@ -778,10 +780,6 @@ func (p *Plugin) doOneShot(ctx context.Context) error {
 
 	p.status.SetError(err)
 
-	if p.metrics != nil {
-		p.status.Metrics = p.metrics
-	}
-
```

Comment:
Here we were adding the global metric registry under the decision log status update. This should not be needed as the status plugin already reports those.